### DISCUSSION TITLE
Allow disable test_before_acquire

### DISCUSSION THREAD

**Discussion by Sytten** [SeaQL/sea-orm#2066] [Ideas]
Hey there! Just missing an option on the pool options to set `test_before_acquire` to false.
I am trying to reduce the number of timeouts I have, I am unsure why but my pool is far from exhausted but I sometimes get timeouts on acquire that are very much not normal.

**Comment by tyt2y3** [comment]
Sure, I think this can be quickly added

Ref: https://docs.rs/sqlx/latest/sqlx/pool/struct.PoolOptions.html#method.test_before_acquire

**Reply by tyt2y3** [reply]
Added in 382ffa3054128f80b8500634e97fe9e30a759f63

**Reply by tyt2y3** [reply]
I am releasing it today.
